docs/admin/system_admin/load_balancing.md

```diff
@@ -0,0 +1,71 @@
+    mode    http
+    option  httplog
+    capture request header Host len 64
+    use_backend leofs_backend
+
+backend leofs_backend
+    mode    http
+    balance roundrobin
+    option  httpchk GET /_leofs_adm/ping HTTP/1.1\r\nHost:\ s3.amazonaws.com
+    timeout http-keep-alive 4500
+
+    server    gw0 gw0.lan:8081 weight 1 maxconn 10000 check
+    server    gw1 gw1.lan:8081 weight 1 maxconn 10000 check
+    server    gw2 gw2.lan:8081 weight 1 maxconn 10000 check
+    server    gw3 gw3.lan:8081 weight 1 maxconn 10000 check
+```
+
+## Health Check URL
+
+One of the important options here is `option httpchk`, which makes LB check if gateway is working and can connect to storage servers. Problematic gateways (which refuse connection, reply with an error or don't provide reply fast enough, which is controlled by `timeout check` parameter here) can be automatically disabled / re-enabled this way. The URL for health check is always **/_leofs_adm/ping**, regardless of gateway mode (S3 or REST). For S3 mode, "Host:" header is required as well (any S3 endpoint can be used there). Similar feature should be activated in configuration of any LB.
+
+!!! note "Note: Health Check in REST mode"
+    You can't get an object stored at **/_leofs_adm/ping** as it's shadowed by the Health Check URL so be careful not to put any object at **/_leofs_adm/ping** in REST mode.
+
+
+## Things to check for production systems
+
+Generally, it's best to use latest version of HAProxy (or, at very least, 1.6). This example is suited (and probably doesn't require much tweaking) for people who just need LB for extra reliability. However, please note that running a high load production system would require tweaking of various timeouts, LB / backend and LB / client connection parameters. Depending on clients and type of load, there is no single perfect solution; it requires tweaking, experimentation and performance measurements (most LBs have internal statistics which can help here), besides, often OS-level tweaking is required as well. Here are some important points both for tweaking this configuration to handle high load and for configuring other load balancers.
+
+**Open files limit**: each connection uses an open file (socket), and operating system limits amount of open files for processes. Generally, for users of official Linux packages that are running LeoFS v1.3.8 and higher these limits should be increased automatically. For other cases, please consult documentation for your OS / distro on how to properly increase file limits for processes; it's a good idea to set this limit to high value like 65535 for LeoGateway (usually, default is 1024). Open files limit should be raised for LB as well, it should be bigger than maximum allowed amount of connections to all backend servers plus maximum amount of client connections; in case of HAProxy, it must be set higher than number in global `maxconn` option (e.g. 31000 for this example with `maxconn 30000`).
```

Comment:
To make it more readable, how about like this
```bash
### in case of HAProxy
open files > maxconn # e.g. set open files to 31000 for this example with maxconn 30000 
```
using expressions rather than sentences.